workflow/automation/install_scripts/install_cybershake_fault.py

```diff
@@ -137,17 +137,15 @@ def install_fault(
 
     vs30_file_path = stat_file_path.replace(".ll", ".vs30")
 
-    # this variable has to be empty
-    # TODO: fix this legacy issue, very low priority
-    event_name = ""
-
     # get all srf from source
     srf_dir = simulation_structure.get_srf_dir(root_folder, fault_name)
 
     list_srf = glob.glob(os.path.join(srf_dir, "*_REL*.srf"))
     if len(list_srf) == 0:
         list_srf = glob.glob(os.path.join(srf_dir, "*.srf"))
-
+    median_path = os.path.join(srf_dir, f"{fault_name}.srf")
```

Comment:
Can we make this using Path library?

Reply:
I chose to use raw strings over path to be consistent with the rest of the function. 
I'd like to convert simulation_structure over to Path, and then the rest of the codebase at some point though